ios/pillarwallet/Info.plist

```diff
@@ -114,8 +80,17 @@
 		<string>MaterialCommunityIcons.ttf</string>
 		<string>MaterialIcons.ttf</string>
 		<string>Octicons.ttf</string>
+		<string>Roboto_medium.ttf</string>
+		<string>Roboto.ttf</string>
+		<string>rubicon-icon-font.ttf</string>
 		<string>SimpleLineIcons.ttf</string>
 		<string>Zocial.ttf</string>
+		<string>Fontisto.ttf</string>
+		<string>EuclidCircularB-Bold.otf</string>
+		<string>EuclidCircularB-Light.otf</string>
+		<string>EuclidCircularB-Medium.otf</string>
+		<string>EuclidCircularB-Regular.otf</string>
+		<string>PillarIcons.ttf</string>
```

Comment:
While trying to build in xCode, these font files throw plenty of Multiple Command errors. This only occurs in new build system, works fine in legacy. Not sure if we need to care about this.

Reply:
Interesting, which XCode you're in? We can commit the legacy build setting to make it easier. Does it stop during build process or just throws error, but runs the app successfully?

Reply:
11.2.1, it stops the build. This might be something wrong with my local setup, so it would be great if someone else tried this too

Reply:
Please check https://github.com/pillarwallet/pillarwallet/compare/fix/pure-react-native...fix/pure-rn-ios-further-work
More specifically included changes in commits 7b6a55f 780abd1